.github/actions/ci/generate-test-matrix/action.yml

```diff
@@ -45,8 +45,10 @@ runs:
 
             "mauilib_MacOS;macOS;unomauilib;"
             "unolib_MacOS;macOS;unolib;"
+            "unolibNative_MacOS;macOS;unolib;--renderer native"
             "mauilib_Windows;Windows;unomauilib;"
             "unolib_Windows;Windows;unolib;"
```

Comment:
I'm curious why this existing test wasn't failing prior to this change when attempting to be built for WASM?

@jeromelaban it should have been failing, no?

Or is this specifically just creating an unolib project but not incorporating it as part of another Uno application project?

Do we have a way to easily generate that kind of application? like an unoapp with an unolib inside?

Reply:
This is only a "new lib", not a new project with a library dependency. It's a scenario that is not covered by the test matrix yet.